engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorImpl.java

```diff
@@ -772,11 +757,31 @@ else if ( isIndexable ) {
 		}
 	}
 
+	private ValueContext<?, Object> buildNewLocalExecutionContext(ValueContext<?, Object> valueContext, Object value) {
```

Comment:
Yes, no change but I needed to use the exact same code a few lines above so I extracted it.